news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,101 @@
+
+The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+A month later on December 15th, touch screen scores had their pp reduced, touch screen detection was introduced, and a new touch screen "Touch Device" mod was added, which would reduce the amount of pp gained from jumps on any map.
+
+A unique usage of touch screen nowadays is using it exclusively as an aiming device, while still tapping with the keyboard. For an example of that, you can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what that looks like.
+
+## "Rapid trigger" feature on mechanical keyboards (2021)
+
+"Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The GIF below visually represents how rapid trigger works.
+
+![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/rapid_trigger_example.gif)
+
+This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still make inputs. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.
+
+This is still a hotly debated topic, but if you want to get an opinion of a pro player, look no further than [BTMC](https://osu.ppy.sh/users/3171691)'s video on the rapid trigger feature:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cFvdM_0XuAg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+---
+
+This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beat maps in any rhythm game ever.
```

Comment:
```suggestion
This is a fascinating way to look back through the ages and remind ourselves of a different, simpler times. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beat maps in any rhythm game ever.
```

Reply:
i used singular instead of plural so the entire sentence would need to be changed to fit "times"

Reply:
The suggested wording here would be incorrect. I think @venix12 made an oops.

Reply:
oh yeah, sorry thought i removed that one